pkg/kubeclient/client_test.go

```diff
@@ -0,0 +1,13 @@
+package kubeclient
+
+import (
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/kubernetes/fake"
+	"testing"
+)
+
+func TestNewKubeClient(t *testing.T) {
```

Comment:
I get this when running the tests:
```
# k8s.io/client-go/rest
../../../../pkg/mod/k8s.io/client-go@v11.0.0+incompatible/rest/request.go:598:31: not enough arguments in call to watch.NewStreamWatcher
        have (*versioned.Decoder)
        want (watch.Decoder, watch.Reporter)
FAIL    github.com/codeready-toolchain/registration-service/pkg/kubeclient [build failed]
```

Reply:
...but only after I did a `go get -u` and a `go mod tidy`. This refreshes the dependencies. Before that, the test was working for me.